packages/cookies/src/response-cookies.ts

```diff
@@ -74,27 +78,20 @@ export class ResponseCookies {
   /**
    * {@link https://wicg.github.io/cookie-store/#CookieStore-delete CookieStore#delete} without the Promise.
    */
-  delete(...args: [key: string] | [options: Cookie]): this {
+  delete(...args: [key: string] | [options: ResponseCookie]): this {
     const name = typeof args[0] === 'string' ? args[0] : args[0].name
     return this.set({ name, value: '', expires: new Date(0) })
   }
 
   // Non-spec
 
-  /**
-   * Uses {@link ResponseCookies.get} to return only the cookie value.
-   */
-  getValue(...args: [key: string] | [options: Cookie]): string | undefined {
-    return this.get(...args)?.value
-  }
-
   /**
    * Uses {@link ResponseCookies.delete} to invalidate all cookies matching the given name.
    * If no name is provided, all cookies are invalidated.
    */
-  clear(...args: [key: string] | [options: Cookie] | [undefined]): this {
+  clear(...args: [key: string] | [options: ResponseCookie] | []): this {
     const key = typeof args[0] === 'string' ? args[0] : args[0]?.name
-    this.getAll(key).forEach((c) => this.delete(c))
+    if (key) this.getAll(key).forEach((c) => this.delete(c))
     return this
   }
```

Comment:
I wonder if this method is confusing. `response.cookies.clear()` might make people believe they can clear all user cookies. this is not the case. Although, this clearly say "response.cookies.clear" so maybe it's not confusing?

Reply:
maybe we should iterate this in the comment?

Reply:
Made it so that you can call `response.cookies.clear()` which will on invalidate all cookies. :+1: